src/sssom_schema/schema/sssom_schema.yaml

```diff
@@ -356,6 +361,21 @@ slots:
     slot_uri: dcterms:description
     examples:
       - value: "This mapping set was produced to integrate human and mouse phenotype data at the IMPC. It is primarily used for making mouse phenotypes searchable by human synonyms at https://mousephenotype.org/."
+  mapping_set_confidence:
```

Comment:
Since this will be a new slot in SSSOM 1.1, please add:

```yaml
mapping_set_confidence:
  instantiates: sssom:Versionable
  annotations:
    added_in: "1.1"
```

to properly mark the slot as compliant with SSSOM 1.1+.

Reply:
Thanks, done! I need to get used to that!